.github/workflows/release.yml

```diff
@@ -107,25 +154,30 @@ jobs:
           if [ "${{ startsWith(github.ref, 'refs/tags') }}" == "false" ] ; then
             # for testing purposes
             echo "tag=test" >> "${GITHUB_OUTPUT}"
+            echo "latest=test-latest" >> "${GITHUB_OUTPUT}"
```

Comment:
To help with creating tags on the fly and use them even for testing the release in a feature branch